### DISCUSSION TITLE
How can I disable full screen?

### DISCUSSION THREAD

**Discussion by anonimo82** [coder/code-server#5616] [Q&A] [answered]
Hello everyone. 
I have my GUI cut on top and bottom in some situations, how can I prevent the App from hiding the address and the status bars?
Note I'm using Chrome on Android, installed Code Server from Termux/npm

Thanks for any help

**Comment by anonimo82** [comment]
Sorry, I've explained it bad.
I have UI cut by the bar which shows time, carrier and battery status (at the top) and the bar with home and back button (at the bottom).
I don't have the problem if I open Chrome and open the editor, but if I add the page to Home screen and go from there, then I have it.
As in this screenshot 
![Screenshot_20221004-080631_Chrome](https://user-images.githubusercontent.com/38261009/193746558-8a8c93cf-a03c-4658-915c-c8506dc180a3.jpg)

**Comment by anonimo82** [comment]
And this is how it looks when opening with Chrome:
![Screenshot_20221004-080937_Chrome](https://user-images.githubusercontent.com/38261009/193747043-d90e3e72-989b-4c37-9566-9d23f9bc4be1.jpg)

How can I force it to show always this way?

**Comment by jsjoeio** [comment, marked as answer]
> How can I force it to show always this way?

Hmm... I think you'd have to modify code-server's source code. IIRC, this is related to the PWA settings and is called `fullscreen` which hides the address bar when running as a PWA.

I think it is here: https://github.com/microsoft/vscode/blob/74b1f979648cc44d385a2286793c226e611f59e7/resources/server/manifest.json#L6

and these are the valid options: https://superpwa.com/doc/web-app-manifest-display-modes/

Note: you'd either have to edit this directly on your device where code-server is installed _or_ fork, edit, recompile and download again on your device.

**Reply by jsjoeio** [reply]
@anonimo82 oops, updated the valid options link

**Comment by anonimo82** [comment]
Ok, thanks
